src/collective/easyform/actions.py

```diff
@@ -349,12 +351,25 @@ def get_header_info(self, fields, request, context,
     def serialize(self, field):
         """Serializa field to save to XML.
         """
-        if isinstance(field, set) or isinstance(field, list):
+        if isinstance(field, set) or isinstance(field, list) or isinstance(field, tuple):
             list_value = list([str(f) for f in field])
             return dumps(list_value)
+        if isinstance(field, dict):
+            dict_value = {str(key): str(val) for key, val in six.iteritems(field)}
+            return dumps(dict_value)
         if isinstance(field, RichTextValue):
             return field.raw
-        return safe_unicode(field)
+        if isinstance(field, datetime):
+            return field.strftime("%Y/%m/%d, %H:%M:%S")
+        if isinstance(field, date):
+            return field.strftime("%Y/%m/%d")
+        if isinstance(field, timedelta):
+            return str(field)
+        if isinstance(field, int) or isinstance(field, float) or isinstance(field, Decimal) or isinstance(field, bool):
+            return str(field)
+        if isinstance(field, six.string_types) or isinstance(field, six.text_type):
+            return safe_unicode(field)
```

Comment:
this is wrong: `six.string_types` and `six.text_type` is unicode already in python 3 ... I assume you want to check for `six.binary_type` and transform it to unicode?